lib/git.zsh

```diff
@@ -9,14 +9,27 @@ function __git_prompt_git() {
   GIT_OPTIONAL_LOCKS=0 command git "$@"
 }
 
-# Outputs current branch info in prompt format
 function git_prompt_info() {
+  # If we are on a folder not tracked by git, get out.
+  # Otherwise, check for hide-info at global and local repository level
+  if ! $(__git_prompt_git rev-parse --git-dir &> /dev/null) \
+     || [[ "$(__git_prompt_git config --get oh-my-zsh.hide-info 2>/dev/null)" == 1 ]]; then
```

Comment:
This is a breaking change; in OMZ the concept of `git status` is equivalent to `git info`, so the current config key is already OK.

Reply:
Yes @mcornella  it is a breaking change, but please keep in mind that  `git_prompt_info` and `git_prompt_status` display different data at different places in the prompt, and these two functions take considerably different amount of time to process depending no the size of the repository.

On the OpenJDK git repository:

- git_prompt_info takes 0.02s
- git_prompt_status takes 2.63s

So I think it is really worthy the breaking change.

Reply:
Alright, I can see it being merged.